test/models/subscription_status_test.rb

```diff
@@ -41,6 +41,11 @@ def test_to_status_unsubscribed_hypervisor
       assert_equal Katello::SubscriptionStatus::UNSUBSCRIBED_HYPERVISOR, status.to_status
     end
 
+    def test_hypervisor_sca_enabled
```

Comment:
Does this test pass without your code change in place?

Reply:
I will comment it out and let you know :)

Reply:
BTW the reason I am asking is because it doesn't appear that this test is using a simple content access org